test/rubocop/cop/minitest/refute_includes_test.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RefuteIncludesTest < Minitest::Test
+  def setup
+    @cop = RuboCop::Cop::Minitest::RefuteIncludes.new
+  end
+
+  def test_registers_offense_when_using_refute_with_include
+    assert_offense(<<~RUBY, @cop)
+      class FooTest < Minitest::Test
+        def test_do_something
+          refute(collection.include?(actual))
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer using `refute_includes(collection, actual)` over `refute(collection.include?(actual))`.
+        end
+      end
+    RUBY
+
+    assert_correction(<<~RUBY, @cop)
+      class FooTest < Minitest::Test
+        def test_do_something
+          refute_includes(collection, actual)
+        end
+      end
+    RUBY
+  end
+
+  def test_registers_offense_when_using_refute_with_include_and_message
+    assert_offense(<<~RUBY, @cop)
+      class FooTest < Minitest::Test
+        def test_do_something
+          refute(collection.include?(actual), 'the message')
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer using `refute_includes(collection, actual, 'the message')` over `refute(collection.include?(actual), 'the message')`.
+        end
+      end
+    RUBY
+
+    assert_correction(<<~RUBY, @cop)
+      class FooTest < Minitest::Test
+        def test_do_something
+          refute_includes(collection, actual, 'the message')
+        end
+      end
+    RUBY
+  end
+
+  def test_registers_offense_when_using_refute_with_include_and_variable_message
```

Comment:
Same as with the other PR - I don't see a point in having separate tests for a variable message and a constant message. Relying on the param count should be enough here.